tools/size.py

```diff
@@ -118,4 +118,4 @@ def compare(t1, t2, target):
 
 
 if __name__ == '__main__':
-    compare("GCC_CR", "LPC1768")
+    compare("ARM", "GCC_ARM", "LPC1768")
```

Comment:
Wat. Why is LPC1768 even a parameter?

Reply:
This is an old script, I'm not 100% sure if this is used anymore. Maybe in the online compiler? So I just tried to get the parameters right. I think revisiting this script in the future would be a good idea though.